openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "./node_modules/@openapitools/openapi-generator-cli/config.schema.json",
```

Comment:
Where is this?

Reply:
Good question, I don't know but this is autogenerated. Personally, the important part of this file is the generator version.